code/datums/supply_packs/food.dm

```diff
@@ -84,13 +213,34 @@
 					/obj/item/reagent_container/food/snacks/grown/wheat,
 					/obj/item/reagent_container/food/snacks/grown/wheat,
 					)
-	cost = 10
+	cost = 7
 	containertype = /obj/structure/closet/crate/freezer
 	containername = "\improper Wheat crate"
 	group = "Food"
 
+/datum/supply_packs/exoticplants
+	name = "exoticplants"
```

Comment:
In all languages that use the Latin alphabet, a space is required between words.

Reply:
i fix those by putting a space in name and _ in object name